cobalt/configuration/configuration.h

```diff
@@ -0,0 +1,79 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COBALT_CONFIGURATION_CONFIGURATION_H_
+#define COBALT_CONFIGURATION_CONFIGURATION_H_
+
+#include <string>
+
+#include "base/notreached.h"
+#include "starboard/extension/configuration.h"
+
+namespace base {
+template <typename T>
+struct DefaultSingletonTraits;
+}
+
+namespace cobalt {
+namespace configuration {
+
+enum UserOnExitStrategy {
```

Comment:
nit: I'd put this enum (needs to be changed to enum class BTW) inside `Configuration`, because it's only used by clients of that class (and to communicate with that class, nothing else). WDYT?